src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/prisoneroffendersearch/PrisonerOffenderSearchGatewayTest.kt

```diff
@@ -108,42 +106,13 @@ class PrisonerOffenderSearchGatewayTest(
         """.trimIndent(),
       )
 
-      val response = prisonerOffenderSearchGateway.getPersons("Obi-Wan", null, null, null)
+      val response = prisonerOffenderSearchGateway.getPersons("Obi-Wan", null, null)
 
       response.data.count().shouldBe(1)
       response.data.first().firstName.shouldBe("Obi-Wan")
       response.data.first().lastName.shouldBe("Kenobi")
     }
 
-    it("returns person(s) when searching on hmppsId only") {
```

Comment:
Was this removed on purpose?

Reply:
Yes, we have excluded pnc search via prisoner, as it is not a search term used by prisoner